src/isInputValid.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { EXPECTED_NUMBER_LENGTH } = require('./constants');
+
+function isInputValid(userInput) {
+  return typeof +userInput === 'number'
+    && [...new Set(userInput.split(''))].length === EXPECTED_NUMBER_LENGTH;
```

Comment:
But would be easier to read if you create variable for this `[...new Set(userInput.split(''))]`